### ISSUE TITLE
Migrate to Github Actions

### ISSUE THREAD

**Issue by cristim** [LeanerCloud/ec2-instances-info#18]
"Please be aware travis-ci.org will be shutting down in several weeks, with all accounts migrating to travis-ci.com. Please stay tuned here for more information."

**Comment by cristim** [comment]
There's also this failure in the latest builds
```
go-bindata: not found
package github.com/go-bindata/go-bindata/go-bindata
	imports github.com/go-bindata/go-bindata/v3: cannot find package "github.com/go-bindata/go-bindata/v3" in any of:
	/home/travis/.gimme/versions/go1.9.linux.amd64/src/github.com/go-bindata/go-bindata/v3 (from $GOROOT)
	/home/travis/gopath/src/github.com/go-bindata/go-bindata/v3 (from $GOPATH)
Makefile:24: recipe for target 'generate-bindata' failed
make: *** [generate-bindata] Error 1
The command "make" exited with 2.
Done. Your build exited with 1.
```

**Comment by cristim** [comment]
Fixed in #19
